tox.ini

```diff
@@ -2,45 +2,70 @@
 envlist =
     check
     docs
-    minimal
-    {py311,py312,py313}{,-coverage}
+    {minimal,nonumba,alltests}{,-coverage}
```

Comment:
do not hesitate to suggest better names for these. This will also be a "workflow breaking change" and have `arviz-stats` differ from the other two repos, but I didn't know how to conciliate this extended configuration with the `py3XY` job names. If we like `alltests` or if we prefer `test` for the recommended full-fledged test suite we can adapt the other two to follow a similar pattern (`tox -e alltests` instead of `py3XY` depending on the python you are in)